frontend/src/app/register/register.component.ts

```diff
@@ -21,58 +21,164 @@ export class RegisterComponent implements OnInit {
     label: 'show',
     type: 'password'
   }
-
+  mode: 'registerStep1' | 'registerStep2' = 'registerStep1';
   months = [
-    'January',
-    'February',]
+    {
+      value: '0',
+      label: 'January'
+    },
+    {
+      value: '1',
+      label: 'February'
+    },
+    {
+      value: '2',
+      label: 'March'
+    },
+    {
+      value: '3',
+      label: 'April'
+    },
+    {
+      value: '4',
+      label: 'May'
+    },
+    {
+      value: '5',
+      label: 'June'
+    },
+    {
+      value: '6',
+      label: 'July'
+    },
+    {
+      value: '7',
+      label: 'August'
+    },
+    {
+      value: '8',
+      label: 'September'
+    },
+    {
+      value: '9',
+      label: 'October'
+    },
+    {
+      value: '10',
+      label: 'November'
+    },
+    {
+      value: '11',
+      label: 'December'
+    }
+  ]
   years = [
     '2020',
     '2021',
-    '2022',]
+    '2022',
+  ]
   days = [
```

Comment:
I should move this constant from here, what is better to save them in mongo or create file for constants?

Reply:
Why do you use constants at all? Can you get current year -3 before dinamically?